anycloud/node/src/datastore.ts

```diff
@@ -34,31 +34,35 @@ export class DataStore {
     if (!dsKey) return undefined;
     if (this.isLocal) {
       if (dsKey in this.localDS) {
-        const val = this.localDS[dsKey]; 
-        if (typeof(val) === 'object') {
-          return JSON.stringify(val);
-        } else {
-          return `${val}`;
-        }
+        const val = this.localDS[dsKey];
+        return JSON.stringify(val);
       } else {
         return undefined;
       }
     }
     try {
-      return await this.request('GET', `${this.ctrlPortUrl}get/${dsKey.toString()}`);
+      return JSON.stringify(await this.request('GET', `${this.ctrlPortUrl}get/${dsKey.toString()}`));
     } catch (e) {
       return e;
     }
   }
 
   async set(dsKey: string, dsValue: any): Promise<string> {
```

Comment:
While we use the `fail/ok` strings for the HTTP responses, this could be a `Promise<boolean>` instead and it's just `return false;` or `return true;`, which is probably going to be less confusing in JS. (The "boolean-equivalent" in the REST API was the response code of 200/400.)